[New Post] Add summary notification settings (#137)
hackclub/hcb-engr: content/posts/summary-notification-settings/post.mdx

import post from '@/components/Post';
export default post({
  title: "🔔 Toggle monthly donation and follower summary emails",
  slug: "summary-notification-settings",
  category: "improvement",
  tags: ["notifications"],
  authors: ["luke"],
  date: new Date("2026-04-14"),
});

You can now toggle whether you receive monthly donation and follower summary emails from your [notification settings](https://hcb.hackclub.com/my/settings/notifications).

> These toggles apply to all organizations you are a member of and only for yourself. More customization coming soon!

Reviewer: ```suggestion
These toggles apply to all organizations you are a member of, and only affect your own account. More customization coming soon!
```
